templates/tfengine/components/devops/main.tf

```diff
@@ -50,36 +50,30 @@ module "project" {
   source  = "terraform-google-modules/project-factory/google"
   version = "~> 10.2.2"
 
-  name            = "{{.project.project_id}}"
+  name            = var.project.project_id
   {{- if eq .parent_type "organization"}}
-  org_id          = "{{.parent_id}}"
+  org_id          = var.parent_id
   {{- else}}
   org_id          = ""
-  folder_id       = "{{.parent_id}}"
+  folder_id       = var.parent_id
   {{- end}}
-  billing_account = "{{.billing_account}}"
+  billing_account = var.billing_account
   lien            = {{get . "enable_lien" true}}
```

Comment:
This `enable_lien` is not specified in schema since it's not a `data` variable at config level. So I don't know if we should create a terraform variable for these type of cases

Reply:
Didn't know it was removed. I set it to true and same for `project` recipe. Even though it's not strictly part of the PR but I considered it convenient. Let me know if I should remove `enable_lien` for `project` recipe in a separate future PR